.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,250 @@
+      artifact_name: pdf-doc
+
+  build:
+    name: Build (XPRESS + OR-Tools=${{ matrix.ortools-xpress }})
+    env:
+      XPRESSDIR: ${{ github.workspace }}/xpressmp813
+      ORTOOLSDIR: ${{ github.workspace }}/or-tools
+
+
+    runs-on: ${{ matrix.os }}
+    needs: doc_generation
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        test-platform: [ubuntu-20.04]
+        ortools-xpress: [ON, OFF]
+
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get release
+      id: get_release
+      uses: bruceadams/get-release@v1.2.0
+      if: ${{ env.IS_RELEASE == 'true' }}
+
+    - name: ccache
+      uses: hendrikmuhs/ccache-action@v1
+      if: ${{ env.IS_PUSH == 'true' }}
+
+    - name: Install libraries
+      run: |
+           sudo apt-get update
+           sudo apt-get install uuid-dev libwxgtk3.0-gtk3-dev
+           sudo apt-get install libboost-test-dev
+
+    - name: Read antares-deps version
+      id: antares-deps-version
+      uses: notiz-dev/github-action-json-property@release
+      with:
+        path: 'antares-deps-version.json'
+        prop_path: 'antares_deps_version'
+
+
+    - name: Download pre-compiled librairies
+      uses: ./.github/workflows/download-extract-precompiled-libraries-tgz
+      with:
+        antares-deps-version: ${{steps.antares-deps-version.outputs.prop}}
+        os: ${{matrix.os}}
+        os-full-name: Ubuntu-20.04
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.x
+
+    - name: Install dependencies
+      run: |
+          python -m pip install --upgrade pip
+          pip3 install -r src/tests/examples/requirements.txt
+
+    - name: Init submodule
+      run: |
+          git submodule update --init src/antares-deps
+          git submodule update --init src/tests/resources/Antares_Simulator_Tests
+
+    - name: Download XPRESS if needed
+      if: ${{ matrix.ortools-xpress == 'ON' }}
+      run: |
+        git clone -b master https://${{ secrets.XPRS_USERNAME }}:${{ secrets.XPRS_TOKEN }}@github.com/rte-france/xpress-mp ${{ env.XPRESSDIR }}
+        ls -la ${{ env.XPRESSDIR }}
+
+    - name: Config OR-Tools URL
+      run: |
+          if [ "${{ matrix.ortools-xpress }}" == "ON" ]; then
+            echo "URL_ORTOOLS=https://github.com/rte-france/or-tools/releases/download/v9.2-rte1.3/ortools_cxx_ubuntu-20.04_static_sirius_xprs.zip" >> $GITHUB_ENV
+          else
+            echo "URL_ORTOOLS=https://github.com/rte-france/or-tools/releases/download/v9.2-rte1.3/ortools_cxx_ubuntu-20.04_static_sirius.zip" >> $GITHUB_ENV
+          fi
+
+    - name: Download OR-Tools
+      id: ortools
+      run: |
+            mkdir -p ${{ env.ORTOOLSDIR }} && cd ${{ env.ORTOOLSDIR }}
+            wget -q -O ortools.zip ${{ env.URL_ORTOOLS }}
+            unzip -q ortools.zip
+            rm ortools.zip
+            echo "::set-output name=label::-ortools-xpress"
```

Comment:
Careful, label will be the same in matrix.ortools-xpress == 'ON' and matrix.ortools-xpress == 'OFF', one asset will be overwritten by the other. Make sure to use an `if..else`